topics/proteomics/tutorials/neoantigen-1-fusion-database-generation/tutorial.md

```diff
@@ -0,0 +1,384 @@
+
+# Neoantigen Fusion Database Generation
+
+## Overview of Fusion Neoantigen Database Workflow
+
+The workflow in this tutorial guides users through the generation of a fusion neoantigen database, covering key steps in bioinformatics to identify, filter, and prepare fusion-specific peptides for further immunological study. Below is an overview of each major stage:
+
+### 1. Get Data
+The process begins with the upload and quality assessment of raw sequencing data, which is then uncompressed. This stage sets the groundwork for all subsequent analyses.
+
+### 2. Fusion Detection and Alignment
+RNA sequencing data undergoes alignment to a reference genome using tools like **RNA STAR**, followed by **Arriba** to detect fusion events. These tools identify gene fusions and help characterize the gene segments that combine to form new fusion genes.
+
+### 3. Filtering and Refinement
+After identifying fusions, various filters are applied to remove non-specific or common fusion events using blacklist data and other criteria. This step ensures that only relevant, unique fusion events are retained for neoantigen prediction.
+
+### 4. Peptide Sequence Extraction and Formatting
+Potential neoantigen peptides are extracted from the fusion gene sequences. Using tools such as **Text Reformatting** and **Tabular-to-FASTA**, the data is transformed into formats suitable for further immunological analysis.
+
+### 5. Final Database Formatting
+The workflow concludes by applying regex adjustments and formatting functions to standardize the output, creating a database of potential fusion neoantigens.
+
+### Summary
+This workflow provides a structured approach to preparing fusion neoantigen data for downstream applications, such as immunotherapy research, by making fusion-derived peptides accessible in a database for experimental or clinical exploration.
+
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://zenodo.org/records/14365542/files/human_reference_genome.fasta
+>    https://zenodo.org/records/14365542/files/human_reference_genome_annotation.gtf
+>    https://zenodo.org/records/14365542/files/RNA-Seq_Reads_1.fastqsanger.gz
+>    https://zenodo.org/records/14365542/files/RNA-Seq_Reads_2.fastqsanger.gz
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets
+> 4. Check that the datatype
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add to each database a tag corresponding to ...
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+# Import Workflow
+
+
+> <hands-on-title>Running the Workflow</hands-on-title>
```

Comment:
Since you take the students through all the steps of the workflow one by one manually, I have moved these workflow run boxes to the ends of the tutorials, under a header of "Rerunning on your own data"